gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,582 @@
+  Each worker of the DaemonSet mentioned below references this ResourceClaimTemplate to ensure they start up IMEX daemons associated with the appropriate ComputeDomain.
+
+- A DaemonSet. 
+
+  Depending on your workload requirements, this DaemonSet forms one (or more) IMEX domains by running IMEX daemons on the set of nodes in a ComputeDomain that requires them.
+  When your workload is deployed, these daemons "follow" the workload pods to the nodes where they have been scheduled. 
+  Through DRA, these daemons are guaranteed to be fully up and running before the workload pods that triggered their creation are allowed to run.
+
+As workload pods that reference a CustomDomain ResourceClaimTemplate, get scheduled they trigger the NVIDIA DRA Driver for GPUs to request access to the same IMEX channel on whatever node they land on. 
+
+Once scheduled to a node, the NVIDIA DRA Driver for GPUs adds a Node label for the ComputeDomain to the node where the workload has been scheduled to indicate the node is part of that ComputeDomain.
+This label is used as a NodeSelector on the DaemonSet mentioned above to trigger the scheduling of its pods to specific nodes.
+
+The addition of this Node label triggers the DaemonSet to deploy an IMEX daemon to that node and start running it.
+
+When all daemons have been fully started, the NVIDIA DRA Driver for GPUs unblocks each worker, injects its IMEX channel into the worker and allows it to start running.
+
+Once all workloads running in a ComputeDomain have run to completion, the label gets removed even if the ComputeDomain itself hasn't been deleted yet.
+This allows these nodes to be reused for other ComputeDomains.
+
+Node and Pod Affinity Strategies
+=================================
+
+A ComputeDomain isn't strictly about MNNVL—it's about running workloads across a group of compute nodes. 
+This means even if some nodes are not MNNVL capable, they can still be part of the same ComputeDomain.
+You must apply NodeAffinity and PodAffinity rules to make sure your workloads run on MNNVL capable nodes.
+
+For example you could set PodAffinity with a required topologyKey set to ``nvidia.com/gpu.clique`` when you require all workloads deployed into the same NVLink domain, but don't care which one. 
+Or use a preferred topologyKey set to ``nvidia.com/gpu.clique`` for workloads to span MNNVL domains but want them packed as tightly as possible. 
+
+Create a CustomDomain and run a workload
+========================================
+
+The example manifest below, ``imex-channel-injection.yaml``, shows the creation of a CustomDomain, ``imex-channel-injection``, and a workload pod referencing the ResourceClaimTemplate, ``imex-channel-0``:
+
+.. literalinclude:: ./manifests/input/imex-channel-injection.yaml
+  :language: yaml
+
+Apply the manifest.
+
+.. code-block:: console
+
+  $ kubectl apply -f imex-channel-injection.yaml
+
+
+#. Optional: View the ComputeDomain resource.

```

Comment:
Shouldn't we also view the pod logs of the workload (which verifies that the imex channel device node was injected)?

Reply:
Added a step to view the pod logs